plugins/kubernetes/app/views/kubernetes_release_groups/new.html.erb

```diff
@@ -37,7 +37,7 @@
               <%= build.project.name %> - <%= build.label %>
             </p>
           <% elsif @project.builds.any? %>
-            <%= form.select :build_id, @project.builds.order('id desc').limit(20).map { |b| [b.label, b.id] } %>
+            <%= form.select :build_id, @project.builds.order('id desc').limit(20).map { |b| [b.label, b.id] }, {}, { class: 'form-control' } %>
```

Comment:
Okay, I'll make a follow-up PR to bring the Kubernetes plugin up to date with those changes.